chord_metadata_service/restapi/utils.py

```diff
@@ -367,7 +367,7 @@ def compute_binned_ages(individual_queryset, bin_size: int) -> list[int]:
     return binned_ages
 
 
-def get_age_numeric_binned(individual_queryset, bin_size: int) -> dict:
+def get_age_numeric_binned(individual_queryset, bin_size: int) -> Dict:
```

Comment:
this change isnt needed, returning a `dict` as a type should work in all versions of python we use